src/main/java/com/playmonumenta/networkrelay/RabbitMQManagerAbstractionBukkit.java

```diff
@@ -60,4 +60,9 @@ public void sendDestOfflineEvent(String dest) {
 		DestOfflineEvent event = new DestOfflineEvent(dest);
 		Bukkit.getPluginManager().callEvent(event);
 	}
+
+	@Override
+	public void sendNetworkMessage(String destination, String channel, JsonObject data, com.rabbitmq.client.AMQP.BasicProperties properties) throws Exception {
```

Comment:
Is this the same override on most/all versions? Why not make it the default and override where needed?